web/gds-admin-ui/src/pages/app/dashboard/Status.js

```diff
@@ -91,30 +93,28 @@ const Status = ({ statuses }) => {
                             <Col xs={4}>
                                 <i className="mdi mdi-progress-question text-warning mt-3 h4"></i>
                                 <h4 className="fw-normal">
-                                    <span>{statusPercents()[STATUS.PENDING_REVIEW] + '%'}</span>
+                                    <span>{statusPercents()[STATUS.PENDING_REVIEW] || 0 + '%'}</span>
                                 </h4>
                                 <p className="text-muted mb-0 fs-6">Pending</p>
                             </Col>
 
                             <Col xs={4}>
                                 <i className="mdi mdi-alert-octagram text-danger mt-3 h4"></i>
                                 <h4 className="fw-normal">
-                                    <span>{statusPercents()[STATUS.REJECTED] + '%'}</span>
+                                    <span>{statusPercents()[STATUS.REJECTED] || 0 + '%'}</span>
```

Comment:
Looks like this is unintentionally stripping off the percentages, I'm not a JS expert but do we need to add some parentheses here?

![Screen Shot 2022-02-07 at 9 28 07 AM](https://user-images.githubusercontent.com/42919891/152818825-d4adb8b0-25fe-4c32-a1b5-92a630c7f5b3.png)

Reply:
@pdeziel good catch 👍  thanks. I added the missing percent symbols.